.github/workflows/java.yaml

```
@@ -0,0 +1,39 @@
+name: Java CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
```

Jeres main/master branch hedder "master" og ikke "main". Forvirringen er opstået fordi git og GitHub ikke er enige om det skal hedde det ene eller det andet. Så hvis ens repository er blevet født på ens egen computer, hedder den master, og hvis den er født på GitHub, hedder den main. (GitHub har hovedkontor i San Fransisco, så de er max woke og har derfor gjort oprør mod master/slave-konnotationer.)

```suggestion
on:
  push:
    branches:
      - master
  pull_request:
    branches:
      - master
```